packages/form-state-manager/src/types/any-object.d.ts

```diff
@@ -1,3 +1,7 @@
+export interface AnyBooleanObject {
+  [key: string]: any;
```

Comment:
should be `:boolean` :wink:

Reply:
..
...
....
Yep... it was made to test you. :smile: